### ISSUE TITLE
Add MergeRequest.mergeError field

### ISSUE THREAD

**Issue by lesiak** [gitlab4j/gitlab4j-api#444]
There is an unmapped `merge_error` field on merge request,
Example from `GET /projects/:id/merge_requests/:merge_request_iid` endpoint
Tested on  Gitlab 12.2.5-ee

```
{
  "id": 230722,
  "iid": 125,
  "project_id": 20639,
  ...
  "merge_error": "Merge request is not mergeable",
  "user": {
    "can_merge": true
  },
  "approvals_before_merge": null
}
```

The same field is also exposed on the merge request webhook, under `object_attributes`.
The `EventMergeRequest.mergeError` was already added with https://github.com/gitlab4j/gitlab4j-api/commit/45ff580c7ddafb6300fc3d413e7289f9758213ca#diff-a564495b369a90174369a502c226b2e6

**Comment by gmessner** [comment]
@lesiak 
I've released GitLab4J-API 4.12.10 with your PR in it.  Please verify and close this issue when you have a chance.

**Comment by gmessner** [comment]
@lesiak 
Since this has been released I am going to go ahead and close it, if you find any issues with it feel free to re-open this issue.
